config/routes.rb

```diff
@@ -2,7 +2,10 @@
   mount_devise_token_auth_for 'User', at: 'auth'
   
   api_version(:module => "api/v1", path: { value: 'api/v1' }, defaults: { format: :json }) do
-    resources :books,  only: %i[index show] 
-  end
+    resources :books,  only: %i[index show]
 
+    resources :users, :except => %i[show index new create edit update destroy] do
```

Comment:
Perfect! Another way to do this is:

```suggestion
    resources :users, only: [] do
```